01-Fundamentals-part-2/assignments2.js

```diff
@@ -21,4 +21,24 @@ console.log("\n");
 console.log("Testing Data 2:");
 avgDolphins = calcAverage(85, 54, 71);
 avgKoalas = calcAverage(23, 34, 27);
-checkWinner(avgDolphins, avgKoalas);
+checkWinner(avgDolphins, avgKoalas);*/
+
+function calcTip(billValue) {
+  if (billValue > 50 || billValue < 300) {
+    //console.log(`The tip is ${billValue * 0.15}$`);
+    return billValue * 0.15;
+  } else if (billValue <= 50 || billValue >= 300) {
+    //console.log(`The tip is ${billValue * 0.2}$`);
+    return billValue * 0.2;
+  } else {
+    console.log("Error! Check your bill!!!");
```

Comment:
if the condition in the first if is not met, then you return 20% as tip. Thus `else if` is redundant in this case.